charts/openshift-metering/values.yaml

```diff
@@ -1,11 +1,14 @@
 global:
   ownerReferences: []
+  useIPV6Networking: false
 
 unsupportedFeatures:
   enableHDFS: false
 
 disableOCPFeatures: false
 
+useIPV6Networking: false
```

Comment:
Hmm, this is being rendered weird for me as it looks like you're adding this to the values.yaml file but that's already present upstream: https://github.com/operator-framework/operator-metering/blob/master/charts/openshift-metering/values.yaml#L9